src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -197,7 +198,8 @@ struct CharacteristicEvolution {
       tmpl::transform<typename metavariables::cce_scri_tags,
                       tmpl::bind<::Actions::MutateApply,
                                  tmpl::bind<CalculateScriPlusValue, tmpl::_1>>>,
-      record_time_stepper_data_and_step>;
+      record_time_stepper_data_and_step,
+      Actions::SendNextTimeToGH<CharacteristicEvolution<Metavariables>>>;
```

Comment:
Can this also be earlier?

Reply:
I'd like to make the order of the self-start list roughly the same as the `extract_action_list`. The location shouldn't matter too much for the self-start phase. We can think about this as we proceed.